apps/src/templates/teacherHomepage/SectionsTable.jsx

```diff
@@ -0,0 +1,151 @@
+import React, { PropTypes } from 'react';
+import color from "@cdo/apps/util/color";
+import i18n from '@cdo/locale';
+
+const styles = {
+  table: {
```

Comment:
Do we have enough tables in React that we should have some common styling?

(This is probably a big question with a lot of ramifications.  Maybe @Bjvanminnen knows of prior discussion? :))